README.md

```diff
@@ -138,10 +415,46 @@ In order to run the account-service in development mode without containers, you
 
 **Note:** Reference `.vscode/launch.json` for more details on the debug configurations and apply the concepts to your preferred debugger.
 
-## Architecture
 
-The account-service is a NestJS application that is split into two main parts: the API and the Worker.
+<p align="right">(<a href="#-table-of-contents">back to top</a>)</p>
 
-The API is responsible for handling incoming HTTP requests and the Worker is responsible for processing jobs that require blockchain interaction.
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+- [Contributing Guidelines](./CONTRIBUTING.md)
+- [Open Issues](https://github.com/AmplicaLabs/acccount-service/issues)
+
+<p align="right">(<a href="#-table-of-contents">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support
+
+If you like this project...loom at the label for `good-first-issue`
```

Comment:
What's "loom"? 😄 
Also, maybe some word-smithing here... maybe "If you would like to explore contributing bug fixes or enhancements, issues with the label `good-first-issue` can be a good place to start."

Reply:
@JoeCap08055 I like it. Thinking we should change the wording in the template.